inferelator_ng/design_response_R.py

```diff
@@ -3,14 +3,11 @@
 """
 
 import os
-import subprocess
```

Comment:
I think it's harmless as well. It looks like the cause might be that the exact same file was modified with the same changes in a different git commit, https://github.com/simonsfoundation/inferelator_ng/pull/13/commits/4125a503ccca800e676a525adb1bf0684f032d9a , which happens to have a later timestample